tests/unit_tests/scheduler/test_lsf_driver.py

```diff
@@ -32,61 +36,518 @@
 
 
 @pytest.fixture
-def mock_bjobs(tmp_path):
-    script = "#!/usr/bin/env python3" + dedent(
-        """
-           import datetime
-           import json
-           import os.path
-           import sys
+def mock_bsub(tmp_path):
+    script_path = tmp_path / "mock_bsub"
+    script_path.write_text(
+        "#!/usr/bin/env python3"
+        + dedent(
+            """
+                import sys
+                import time
+                import random
+                run_path = sys.argv[-1]
+                with open("job_paths", "a+", encoding="utf-8") as jobs_file:
+                    jobs_file.write(f"{run_path}\\n")
+
+                # debug purposes
+                with open("bsub_log", "a+", encoding="utf-8") as f:
+                    f.write(f"{' '.join(sys.argv)}\\n")
+
+                time.sleep(0.5)
+
+                if "exit.sh" in sys.argv:
+                    exit(1)
+
+                if "gargled_return.sh" in sys.argv:
+                    print("wait,this_is_not_a_valid_return_format")
+                else:
+                    _id = str(random.randint(0, 10000000))
+                    print(f"Job <{_id}> is submitted to default queue <normal>.")
+            """
+        )
+    )
+    os.chmod(script_path, 0o755)
+
+
+@pytest.fixture
+def mock_bkill(tmp_path):
+    script_path = tmp_path / "mock_bkill"
+    script_path.write_text(
+        "#!/usr/bin/env python3"
+        + dedent(
+            """
+                import sys
+                import time
+                import random
+                job_id = sys.argv[-1]
+                with open("job_ids", "a+", encoding="utf-8") as jobs_file:
+                    jobs_file.write(f"{job_id}\\n")
+
+                time.sleep(0.5)
+
+                if job_id == "non_existent_jobid":
+                    print(f"bkill: jobid {job_id} not found")
+                    exit(1)
+            """
+        )
+    )
+    os.chmod(script_path, 0o755)
+
+
+class MockStateHandler:
+    id = "SUBMITTED"
+
+
+class MockRunArg:
+    runpath = "/usr/random/ert_path"
+
+    def iens():
+        return 0
+
+
+class MockQueueableRealization:
+    run_arg = MockRunArg()
+
+
+class MockRealizationState:
+    _state = "SUBMITTED"
+    _verified_killed = False
+    realization = MockQueueableRealization()
+    current_state = MockStateHandler()
+
+    def verify_kill(self):
+        self._verified_killed = True
+        print("Realization was verified killed")
+
+    def accept(self):
+        self.current_state.id = "PENDING"
+
+    def start(self):
+        self.current_state.id = "RUNNING"
+
+    def runend(self):
+        self.current_state.id = "DONE"
+
+
+def create_fake_bjobs_result(dir: str, job_id: str, status: str):
+    # print("JOBID USER STAT QUEUE FROM_HOST EXEC_HOST JOB_NAME SUBMIT_TIME")
+    Path(dir / "mocked_result").write_text(
+        f"{job_id}\tpytest\t{status}\ttest_queue\thost1\thost2\ttest_job\t{str(datetime.now())}"
```

Comment:
wild guess, the real bjobs will not use the same timeformat as `str(datetime.now())`